tests/Integration/Behaviour/Features/Context/Domain/OrderFeatureContext.php

```diff
@@ -563,9 +563,21 @@ public function checkOrderInvoicesCount(string $orderReference, int $expectedCou
     public function editProductsToOrderWithFollowingDetails(string $productName, string $orderReference, TableNode $table)
     {
         $orderId = SharedStorage::getStorage()->get($orderReference);
-        $productOrderDetail = $this->getOrderDetailFromOrder($productName, $orderReference);
         $data = $table->getRowsHash();
 
+        $order = new Order($orderId);
```

Comment:
What was the problem with `getOrderDetailFromOrder`? It's been used for a long time now and seemed to serve its purpose